.github/CODEOWNERS

```diff
@@ -0,0 +1,48 @@
+# Centrifudge Code Owners
+#
+# Order is important; the last matching pattern takes the most
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*       @branan @mikiquantum @mustermeiszer @offerijns
+
+# Changes to the service of our chain.
+src/*    @mikiquantum
+
+# Changes to pallets
+pallets/*  @branan @mikiquantum @mustermeiszer
+
+## Changes to specific pallets
+### Connectors related logic
+pallets/connectors/* @NunoAlexandre
+
+### Rewards related logic
+pallets/rewards/* @lemunozm
+pallets/liquidity-rewards/* @lemunozm
+
+### Nft-sales related logic
+pallets/nft-sales/* @NunoAlexandre
+
+### Pallet Keystore
+pallets/keystore/* @cdamian
+
+# Changes to libs
+libs/* @branan @mikiquantum @mustermeiszer
+
+## Changes to specific libraries
+### cfg-traits crate
+#### rewards module
+libs/traits/src/rewards.rs @lemunozm
+
+#### ensure module
+libs/traits/src/ops.rs @lemunozm
+
+# Changes to runtime
+runtime/* @branan @mikiquantum @mustermeiszer
+
+## Changes to folders inside runtime
+### Changes to runtime/integration-tests
+runtime/integration-tests/* @mustermeiszer
+
+### Changes to runtime/integration-tests/xcm/
+runtime/integration-tests/xcm/* @NunoAlexandre
```

Comment:
```suggestion
runtime/integration-tests/xcm/* @NunoAlexandre

```

Reply:
I will still keep you in that line, additionally, as this will notify you for review. Wdyt?

Reply:
Sounds good, thanks 🙏